### DISCUSSION TITLE
有什么快速判断每个节点是否可用的办法？

### DISCUSSION THREAD

**Discussion by GongT** [v2fly/v2ray-core#1160] [Q&A]
买了几个机场，都会有很多节点。公用节点还是比较容易被被封的，官方也不会马上换ip什么的，导致订阅列表里有的能用有的不能。

之前用过手机上的v2ray，有个测试真实链接的功能，电脑上有没有什么工具可以实现类似的功能？

手动挨个测试实在太麻烦了（我放路由器上的，每次都得查日志找哪个节点坏了再改配置文件，相当麻烦）

我看v2ray有动态添加删除节点的api，理论上可以做个全自动替换可用节点的工具。。。？

**Comment by arescai** [comment]
https://github.com/2dust/v2rayN/releases 试试这个

**Reply by GongT** [reply]
我在路由器上运行的qwq

**Comment by ghost** [comment]
A simple way is using `curl`, e.g. `curl https://www.google.com/`.

Just write a script to perform automatical tests.
